bodhi/tests/server/services/test_updates.py

```diff
@@ -101,7 +101,7 @@ class TestNewUpdate(BasePyTestCase):
     def mock_getTag(cls, tag, *kwargs):
         if tag == 'f17-build-side-7777':
             return {'maven_support': False, 'locked': False, 'name': 'f17-build-side-7777',
-                    'extra': {'sidetag_user': 'dudemcpants', 'sidetag': True},
+                    'extra': {'sidetag_user': 'guest', 'sidetag': True},
```

Comment:
why did we need to change this ? just curious :)

Reply:
Because of `test_new_rpm_update_from_tag()`.
There the user guest tries to create a new side-tag update and it must succeed. But if the side-tag is owned by dudemcpants, as the mocked getTag replies, the test will fail.

I could modify the test by creating a new 'dudemcpants' user and make it create the update, but that's more work and I'm lazy :-p

Reply:
:+1: